test/fixtures/define-properties/case-1/before.js

```diff
@@ -0,0 +1,5 @@
+const sup = require('define-properties').supportsDescriptors;
```

Comment:
I haven't added support for when the module is imported without directly accessing the `supportsDescriptors` property.

Reply:
I think thats fine for now 👍 If this turns out to be a big missing thing we can always add it later